lib/galaxy/workflow/modules.py

```diff
@@ -353,8 +353,11 @@ def get_data_outputs(self):
                     label = workflow_output['label']
                     if not label:
                         label = "%s:%s" % (order_index, workflow_output['output_name'])
+                    workflow_output_uuid = object()
+                    if workflow_output.get('uuid'):
+                        workflow_output_uuid = workflow_output['uuid']
```

Comment:
You can do the same in one line:
```python
workflow_output_uuid = workflow_output.get('uuid') or object()
```
For consistency, I'd also define here `data_output_uuid` in a similar way.